### PR TITLE
liburing: update to 2.2.

### PR DESCRIPTION
**Pull request by dragon-friend**
#### Testing the changes
- I tested the changes in this PR: briefly

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl (crossbuild)
  - armv7l (crossbuild)
  - armv7l-musl (crossbuild)

### CODE REVIEW

File: srcpkgs/liburing/template [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 revision=1
 build_style=configure
 configure_args="--mandir=/usr/share/man"
 make_build_args="-C src"
 short_desc="Linux-native io_uring I/O access library"
-maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
+maintainer="Dragon Friend <friend-dragon@proton.me>"
 license="LGPL-2.1-only"
 homepage="http://git.kernel.dk/cgit/liburing"
 distfiles="${homepage}/snapshot/${pkgname}-${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/liburing/template:12]: expand homepage and pkgname here

File: srcpkgs/liburing/template [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 pkgname=liburing
-version=2.1
+version=2.2
 revision=1
 build_style=configure
 configure_args="--mandir=/usr/share/man"
 make_build_args="-C src"
 short_desc="Linux-native io_uring I/O access library"
-maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
+maintainer="Dragon Friend <friend-dragon@proton.me>"
```
Comment (classabbyamp) [inline srcpkgs/liburing/template:9]: @ailiop-git can they adopt this from you?

Comment (ailiop-git) [reply srcpkgs/liburing/template:9]: sure, go ahead

**Comment by classabbyamp** [comment]
please add your committing email to your github account

**Comment by dragon-friend** [comment]
all done, thanks for the help with this! :)

**Pull request merged**